core/src/driver/stablex_driver.rs

```diff
@@ -254,11 +244,21 @@ mod tests {
         }
     }
 
+    /// Always returns 1 max gas price cap.
+    fn default_economic_viability() -> Arc<dyn EconomicViabilityComputing + Sync> {
```

Comment:
I'm not a huge fan of this method, since it makes the unit tests a little harder to follow (you have to check the implementation of the method to know what value is being set) and would prefer if that were inlined in the unit test.

Also, if the goal is to simplify construction of this, you could always use `FixedEconomicViabilityComputer` instead of `MockEconomicViabilityComputing`. Alternatively, maybe the `max_gas_price` value could be passed in as a parameter to the method.

Reply:
Good point about using Fixed directly. Will change it to that.